lang/id.po

```diff
@@ -0,0 +1,99 @@
+msgid ""
+msgstr ""
+"Language: id\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n==1 ? 0 : 1);\n"
+
+# A place for you to credit yourself!
+# Put something like "Latin translation by Jvlivs Gaivs\n" here.
+msgid "TRANSLATION_AUTHOR\n"
+msgstr "Diterjemahkan ke Bahasa oleh Habib Fikri\n"
+
+msgid "FREE_PACKAGES_COUNT\n"
+msgid_plural "FREE_PACKAGES_COUNT\n"
+msgstr[0] "%d paket free (dari total %s)\n"
+msgstr[1] "%d paket free (dari total %s)\n"
```

Comment:
Just to make sure - are these the proper plural forms for Indonesian? `msgstr[0]` refers to a single free package, `msgstr[1]` to multiple.

Reply:
Yes, it is. In Bahasa, if a noun prefixes with its amount, the form is the same as its singular form.

Reply:
In that case, we can remove the plural forms, it seems. Please edit the file header like this:
```
Plural-Forms: nplurals=1; plural=0;
```
And then remove `msgstr[1]` from both plural strings, leaving only `msgstr[0]`.